### ISSUE TITLE
[Accessibility][Accordion] Missing visual keyboard focus on the accordion headings/trigger buttons

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1782]
**Steps to Reproduce**
1. Open page in Google Chrome.
2. Use TAB and/or TAB + SHIFT to navigate through the accordion component headings.

**Current Behavior**
When focused, the accordion headings provide no discernible visual indication of focus, because the CSS `outline` property's value is set to `none`. It is difficult for sighted keyboard users to understand where their keyboard focus is and how to interact with their intended control.

**Expected behavior/code**
Actionable/focusable items should provide a visual indicator of focus. When the default focus indicator is removed (e.g. with `outline: none`), it must be replaced with a focus indicator that provides at least a 3:1 contrast ratio with the background colors.

**Environment**
Chrome 92, Windows 10
AEM Core Components v. 2.17  

**Affected Users**
Sighted keyboard users

**WCAG SC**
2.4.7 Focus Visible

**Comment by richardhand** [comment]
@adobe export issue to Jira project SITES
